### ISSUE TITLE
Uncaught exceptions in async methods leave no stacktrace

### ISSUE THREAD

**Issue by LoveIsGrief** [onury/grunt-jasmine-nodejs#31]
**Example:**

```coffeescript
describe 'wtf', ->
  it 'should throw an uncaught exception', (done)->
    setTimeout(->
      throw new Error 'What have I done !??!?!'
    ,100)
```

**Result:**
![no stack trace](https://cloud.githubusercontent.com/assets/2829538/13315937/ed9980f0-dbad-11e5-9327-b3ee22b92a43.png)

**Notes:**
jasmine-contrib/grunt-jasmine-node#21 added a `captureExceptions` option to the tasks which would dump the the stack on such exceptions.

**Comment by onury** [comment]
I don't think [that](https://github.com/venables/grunt-jasmine-node-coverage/commit/5463424d298594e671bd669487a899491f3841a9#diff-d0355f4a765bab8ed3ac0bf9d766f19aR220)'s necessary.
You can achieve the same thin with `grunt --stack` command.
This will output the stack trace for fatal errors (i.e. `uncaughtException`).

I'll add an option to the plugin that will run the task with this flag, shortly.
Thanks.

**Comment by LoveIsGrief** [comment]
IDEA Webstorm and maybe other IDEs don't seem to be able to pass options to the grunt command. That could be a feature request to the respective IDEs too.

Looking forward to your upcoming option.

**Comment by onury** [comment]
Just released v1.5.0
Pls check out the [`traceFatal`](https://github.com/onury/grunt-jasmine-nodejs#tracefatal) option and let me know if this works for you.

plus, there are some new features...

**Comment by LoveIsGrief** [comment]
Thanks @onury ! 

**Results:**
![async fail - good fail](https://cloud.githubusercontent.com/assets/2829538/13374691/71168e34-dd8c-11e5-98f6-297ee7ace45f.png)

:+1:
